browser/ui/tabs/split_view_browser_data.cc

```diff
@@ -229,14 +229,14 @@ void SplitViewBrowserData::TabsWillBeAttachedToNewBrowser(
   }
 
   if (tiles_to_be_attached_to_new_window_.size() > 1) {
-    base::ranges::sort(tiles_to_be_attached_to_new_window_);
-    tiles_to_be_attached_to_new_window_.erase(
-        base::ranges::unique(tiles_to_be_attached_to_new_window_,
-                             [](auto& a, auto& b) {
-                               return a.first == b.first &&
-                                      a.second == b.second;
-                             }),
-        tiles_to_be_attached_to_new_window_.end());
+    std::ranges::sort(tiles_to_be_attached_to_new_window_,
+                      [](const auto& a, const auto& b) { return a < b; });
+    auto to_remove = std::ranges::unique(
```

Comment:
why copmparator was added?

Reply:
Done. I had noticed the comparison was being required but I didn't look much further, but it turns out the the comparison ops were not declared correctly in `TabTile` which was removing support for weak strict ordering.